### ISSUE TITLE
SkyBox Component

### ISSUE THREAD

**Issue by ajlende** [darwin-education/resium#109]
I have a custom skybox that I want to use that I've tried setting via the Scene, Viewer, and CesiumWidget. The skybox is static, so I first tried creating it outside of the render as recommended.

```js
const skyBox = new SkyBox({ sources });
const MyComponent = () => <CesiumWidget skyBox={skyBox} />;
```

When MyComponent was unmounted and remounted, I got an error.

```
DeveloperError: This object was destroyed, i.e., destroy() was called.
Error
    at new DeveloperError (http://localhost:1485/static/app.js:33343:19)
    at SkyBox.throwOnDestroyed (http://localhost:1485/static/app.js:90574:19)
    at updateEnvironment (http://localhost:1485/static/app.js:214114:83)
    at render (http://localhost:1485/static/app.js:214531:9)
    at tryAndCatchError (http://localhost:1485/static/app.js:214552:13)
    at Scene.../node_modules/cesium/Source/Scene/Scene.js.Scene.render (http://localhost:1485/static/app.js:214620:13)
    at CesiumWidget.../node_modules/cesium/Source/Widgets/CesiumWidget/CesiumWidget.js.CesiumWidget.render (http://localhost:1485/static/app.js:275541:25)
    at render (http://localhost:1485/static/app.js:274916:32)
```

The skyBox gets destroyed with the CesiumWidget's destroy method, but the actual skyBox reference remains in a destroyed state causing it to error the next time the component is mounted.

I worked around it by creating the skyBox on the initial render each time and setting it on the Scene, but it seemed that it would be nicer for the skyBox to just be a component to avoid this workaround.

```js
const MyComponent = () => {
  // Create the skyBox on initial render because it gets destroyed on unmount
  // via the CesiumWidget
  const skyBox = useRef();
  useEffect(() => {
    skyBox.current = new SkyBox({ sources });
  }, []);

  // skyBox needs to be set on the Scene because it is a cesium read only
  // property on the CesiumWidget, but not on the Scene.
  return (
    <CesiumWidget>
      <Scene skyBox={skyBox.current} />
    </CesiumWidget>
  );
}
```

**Comment by rot1024** [comment]
You're right. SkyBox component is needed. I'll add SkyBox component soon!

**Comment by rot1024** [comment]
Hi, I added `SkyBox` component and released v1.7.0. Try it!
